frontend/src/config/i18n/de/guidedTour.json

```
@@ -0,0 +1,28 @@
+{
+    "next": "Weiter",
+    "finish": "Abschließen",
+    "back": "Zurück",
+    "mapEditor": {
+        "welcome_title": "Willkommen im Map Editor",
+        "toolboxIntro_title": "Toolbox",
+        "layersIntro_title": "Ebenen",
+        "timelineIntro_title": "Zeitleiste",
+        "baseLayer_title": "Basis Ebene",

```

```suggestion
        "baseLayer_title": "Basis-Ebene",

```
Alternatively:
```suggestion
        "baseLayer_title": "Basisebene",

```

This should be one word afaik. 
@markus2330 Do we already have some sort of standard that says how layers should be addressed?

---

Yes, in German we use your first variant: Basis-Ebene

@badnames thank you, very helpful review!